File: src/quickstart/doc2pdf/java/Doc2Pdf.java
/* 
 * Copyright 2001-2014 Aspose Pty Ltd. All Rights Reserved.
 *
 * This file is part of Aspose.Words. The source code in this file
 * is only intended as a supplement to the documentation, and is provided
 * "as is", without warranty of any kind, either expressed or implied.
 */
  
package quickstart.doc2pdf.java;
import com.aspose.words.*;
public class Doc2Pdf
{
    public static void main(String[] args) throws Exception
    {
        // The path to the documents directory.
        String dataDir = "src/quickstart/doc2pdf/data/";
        // Load the document from disk.
        Document doc = new Document(dataDir + "Template.doc");
        // Save the document in PDF format.
        doc.save(dataDir + "Doc2PdfSave Out.pdf");
    }
}

File: Plugins/Aspose_Words_Java_for_Ruby/lib/asposewordsjavaforruby/doc2pdf.rb
module Asposewordsjavaforruby
  module Doc2Pdf

    def doc_to_pdf()
        data_dir = File.dirname(File.dirname(File.dirname(__FILE__))) + '/data/'
        
        # Open document.
        document = Rjb::import('com.aspose.words.Document').new(data_dir + "Template.doc")
        
        # Save the document in PDF format.
        document.save(data_dir + "Doc2PdfSave Out.pdf") 
    end

  end
end
